tests/ui/UnreadIndicatorsTest.js

```diff
@@ -31,6 +31,14 @@ jest.setTimeout(30000);
 
 jest.mock('../../src/libs/Notification/LocalNotification');
 
+// we need to mock it for the ReportScreen to update its state immediately for tests to pass
```

Comment:
Please be sure to capitalize the first word in every comment. Also, this comment is only a partial explanation of why the mocking is necessary. Can you please improve it so that it also says why the state isn't updating immediately in the first place and why the tests fail when it doesn't update immediately?

Reply:
I just saw that we have a special file for adding the mocks to `react-native`, so I'll move this mock there. Should I add a comment there or maybe just in the PR?

Reply:
Ah, thanks for moving it to the `__mocks__` folder! Can you please add the comment there in the file?